panel/src/config/search.js

```diff
@@ -1,71 +1,71 @@
-
-export default (app) => {
-  const api = async ({endpoint, query, limit, fields, map}) => {
+export default app => {
+  const api = async ({ endpoint, query, limit, fields, map }) => {
     const response = await app.$api.get(endpoint, {
       q: query,
       limit: limit || 10,
-      select: ["id", ...fields, "panelIcon", "panelImage"]
+      select: ["id", ...fields, "panelImage"]
     });
 
+    console.log(["id", ...fields, "panelImage"]);
+
     return response.data.map(item => {
+      console.log(item);
```

Comment:
@distantnative Were these logs intended? Or just used while debugging?

Reply:
Only for debugging -  but this has been fixed already quite some time ago. And by now `search.js` doesn't exist anymore (got replaced by Fiber implementation).